integration-tests/src/test/java/org/web3j/protocol/core/TestnetConfig.java

```diff
@@ -12,22 +12,66 @@
  */
 package org.web3j.protocol.core;
 
+import java.io.IOException;
+import java.math.BigDecimal;
 import java.math.BigInteger;
 import java.util.Arrays;
 
+import org.com.test.contract.Fibonacci;
+
 import org.web3j.abi.EventEncoder;
 import org.web3j.abi.TypeReference;
 import org.web3j.abi.datatypes.Event;
 import org.web3j.abi.datatypes.Uint;
+import org.web3j.crypto.Credentials;
+import org.web3j.protocol.Web3j;
 import org.web3j.protocol.core.methods.request.Transaction;
+import org.web3j.protocol.core.methods.response.TransactionReceipt;
+import org.web3j.tx.TransactionManager;
+import org.web3j.tx.Transfer;
+import org.web3j.tx.gas.ContractGasProvider;
+import org.web3j.utils.Convert;
 
-/** Mordon Testnet Configuration. */
+/** Web3j-Unit Embedded Testnet Configuration. */
 public class TestnetConfig implements IntegrationTestConfig {
 
+    final String blockHash = "0xeba1ae1115682dfa88f7c8c82bbdcb8ce5b599c05d688ce5c715383637199b15";
+    final BigInteger validBlock = BigInteger.valueOf(0);
+    final BigInteger validBlockTransactionCount = BigInteger.valueOf(0);
+    final BigInteger validBlockUncleCount = BigInteger.valueOf(0);
+    final String validAccount = "0xFE3B557E8Fb62b89F4916B721be55cEb828dBd73";
+    final String validPrivateKey =
+            "0x8f2a55949038a9610f50fb23b5883af3b4ecb3c3bb792cbcefbd1542c692be63";
+    private final String validTransactionHash;
+    private final String validContractAddress;
+    private final String validBlockHash;
+    private final BigInteger validBlockNumber;
+    private final BigInteger transactionIndex;
+    private final Fibonacci validDeployedContract;
+
+    TestnetConfig(
+            Web3j web3j, TransactionManager transactionManager, ContractGasProvider gasProvider)
+            throws Exception {
+        this.validDeployedContract =
+                Fibonacci.deploy(web3j, transactionManager, gasProvider).send();
+        TransactionReceipt transactionReceipt = validDeployedContract.getTransactionReceipt().get();
+        this.validContractAddress = validDeployedContract.getContractAddress();
+        this.validTransactionHash = transactionReceipt.getTransactionHash();
+        this.validBlockHash = transactionReceipt.getBlockHash();
+        this.validBlockNumber = transactionReceipt.getBlockNumber();
+        this.transactionIndex = transactionReceipt.getTransactionIndex();
+    }
+
+    @Override
+    public Fibonacci getValidDeployedContract() {
+        return this.validDeployedContract;
+    }
+
     @Override
     public String validBlockHash() {
-        https: // testnet.etherscan.io/block/1627453
-        return "0xd67e59db999c3bd78bd4c2ba54689dba0c372ebcad09c8b9677970f37d64ca46";
+        https:
```

Comment:
I believe this label should be part of the Etherscan URL in the comment, not a Java code label.